handlers/handlers.go

```diff
@@ -181,8 +204,126 @@ func readFindDataset(w http.ResponseWriter, req *http.Request, cfg *config.Confi
 	rend.BuildPage(w, m, "search")
 }
 
+func readDataAggregation(w http.ResponseWriter, req *http.Request, cfg *config.Config, zc ZebedeeClient, rend RenderClient, searchC SearchClient,
+	accessToken, collectionID, lang string, cacheList cache.List, template string,
+) {
+	ctx, cancel := context.WithCancel(req.Context())
+	defer cancel()
+
+	urlQuery := req.URL.Query()
+
+	// get cached census topic and its subtopics
+	censusTopicCache, err := cacheList.CensusTopic.GetCensusData(ctx)
+	if err != nil {
+		log.Error(ctx, "failed to get census topic cache", err)
+		setStatusCode(w, req, err)
+		return
+	}
+
+	// get cached navigation data
+	navigationCache, err := cacheList.Navigation.GetNavigationData(ctx, lang)
+	if err != nil {
+		log.Error(ctx, "failed to get navigation cache", err)
+		setStatusCode(w, req, err)
+		return
+	}
+
+	validatedQueryParams, err := data.ReviewDataAggregationQuery(ctx, cfg, urlQuery, censusTopicCache)
+
+	if err != nil && !errs.ErrMapForRenderBeforeAPICalls[err] {
```

Comment:
Remove the whitespace between function assignment and error handling 
```suggestion
	validatedQueryParams, err := data.ReviewDataAggregationQuery(ctx, cfg, urlQuery, censusTopicCache)
	if err != nil && !errs.ErrMapForRenderBeforeAPICalls[err] {
```